resources/import.py

```diff
@@ -5,25 +5,25 @@
 import os
 from common import *
 
+# for import, we assign the s3 credentials to an environment variable (vs with export, we use the bound service)
+# this allows for us to use a s3 bucket in another environment by just putting the credentials in the env vars.
```

Comment:
for export, the plugin, as of right now, will never know the credentials prior to doing the export. the export app gets the credentials from the bound service. and you can't get the s3 credentials any other way. it has to be through an app that is bound to it.

now by the time we run the import app, we have ran the export app and we were able to get the s3 credentials since we could read the env vars from the bound export app. the plugin saves those credentials locally then when deploying the import app, we inject them into the app. this allows the import app to connect to the s3 bucket without having to bind to it.